lib/db.js

```diff
@@ -418,20 +455,37 @@ module.exports = (
   }
 
   DB.prototype.updateSessionToken = function (token, userAgentString) {
-    log.trace({ op: 'DB.updateSessionToken', uid: token && token.uid })
-
-    if (! token.update(userAgentString)) {
+    log.trace({ op: 'REDIS.updateSessionToken', uid: token && token.uid })
+    if (! token.update(userAgentString) || ! this.redis) {
       return P.resolve()
     }
 
-    return this.pool.post('/sessionToken/' + token.id + '/update', {
-      uaBrowser: token.uaBrowser,
-      uaBrowserVersion: token.uaBrowserVersion,
-      uaOS: token.uaOS,
-      uaOSVersion: token.uaOSVersion,
-      uaDeviceType: token.uaDeviceType,
-      lastAccessTime: token.lastAccessTime
-    })
+    if (this.redis) {
```

Comment:
Not sure if you need the `if` block here because it get caught in the above `if` block.